### PR TITLE
🔄 Sync: Changes from tree-sitter repo

### LINKED ISSUE #66: New commit to tree-sitter/tree-sitter-python
There was a new commit/commits to tree-sitter/tree-sitter-python, please check the newly updated codebase. Here are the recent commits: <br />https://github.com/tree-sitter/tree-sitter-python/commit/29c0d5175170b3f278efe1196843ba3d96c93536

### LINKED ISSUE #68: New commit to tree-sitter/tree-sitter-python
There was a new commit/commits to tree-sitter/tree-sitter-python, please check the newly updated codebase. Here are the recent commits: <br />https://github.com/tree-sitter/tree-sitter-python/commit/d00b1d4016bdaf74e24a1fbddcfd07e08deebcd0

### LINKED ISSUE #69: New commit to tree-sitter/tree-sitter-python
There was a new commit/commits to tree-sitter/tree-sitter-python, please check the newly updated codebase. Here are the recent commits:

### LINKED ISSUE #70: Update: Sync latest changes from TreeSitter to repo
This repository is closely related to Tree Sitter's tree-sitter-python repo. Reviews of the latest commits from that repo should be made to see if we need to update anything in this repository. See the following:

- #66 
- #68 
- #69

### LINKED ISSUE #73: New commit to tree-sitter/tree-sitter-python
There was a new commit/commits to tree-sitter/tree-sitter-python, please check the newly updated codebase. Here are the recent commits: <br />https://github.com/tree-sitter/tree-sitter-python/commit/58f57240834d6b88624e32ad0ab9531d55fb7a5d<br />https://github.com/tree-sitter/tree-sitter-python/commit/6109e4e7bcc59054705f4e05a066804930562993<br />https://github.com/tree-sitter/tree-sitter-python/commit/f132f9a5745187dc5cb3f2c993d12319f6947b3c<br />https://github.com/tree-sitter/tree-sitter-python/commit/bbdac841216a98a43820bdcfb3c3fa5f187327d2<br />https://github.com/tree-sitter/tree-sitter-python/commit/2908cd1366f9e076163f82c1619d034a28cb630e<br />https://github.com/tree-sitter/tree-sitter-python/commit/55d99c927f6fb7f140d1b942034f279d8cdf8901<br />https://github.com/tree-sitter/tree-sitter-python/commit/0e1af55666aa12e3337d006db9c72130bae62989

### PR DESCRIPTION
**Pull request by alxnull**
## 🔄 Sync: Changes from tree-sitter repo

This pull request (PR) is made in reference to: #70

## Description

Sync changes from tree-sitter repo, as mentioned in #70 (closes #70).

- closes #66
- closes #68 
- closes #69 (seems to be empty?)
- closes #73

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

## Testing

* [x] My submission passes the tests?

### CODE REVIEW

**Comment by madiedgar** [comment]
@alxnull thank you so much for this!! I'll look into this today 👍

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "js-beautify": "^1.11.0",
     "prettier": "^1.18.2",
     "prettydiff2": "^2.2.8",
-    "tree-sitter-cli": "^0.16.8",
+    "tree-sitter-cli": "^0.16.9",
```
Comment (madiedgar) [inline package.json:33]: **Line 4-6: the `details` object**

Can you update both attributes within this object to reflect the current synced version of `tree-sitter/tree-sitter-python`? This helps us stay up to date with new commits to their repository. 👍  So for instance, `"latest-commit": "58f57240834d6b88624e32ad0ab9531d55fb7a5d"`. 

(This isn't for line 33, but I can't review the line that needs updating.)

File: queries/highlights.scm [hunk 2]
```diff
@@ -34,12 +34,16 @@
```
Comment (madiedgar) [inline queries/highlights.scm:34]: **Lines 9-24: `Function Calls` and `Builtin Functions`**

Not sure why they went for such a change, but these two sections were reordered to have `Builtin functions` to be before `Function calls`. For consistency sake, can we do the same? 

(again, sorry for the misplaced comment)

**Review by madiedgar — CHANGES_REQUESTED** [review]
Thank you @alxnull for this thorough contribution! Its extremely helpful. 👍  Made a couple of change requests, but other than that this looks great!!

**Comment by alxnull** [comment]
@madiedgar thanks for the feedback! I implemented the changes you suggested.

**Review by madiedgar — APPROVED** [review]

**Pull request merged**